xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-pageobjects/src/main/java/org/xwiki/flamingo/skin/test/po/AttachmentsViewPage.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.flamingo.skin.test.po;
+
+import org.openqa.selenium.By;
+import org.xwiki.test.ui.po.ViewPage;
+
+/**
+ * Represents a page with the attachments pane active.
+ *
+ * @version $Id$
+ * @since 14.7RC1
+ */
+public class AttachmentsViewPage extends ViewPage
```

Comment:
How is this class different from `org.xwiki.test.ui.po.AttachmentsPane`?

Reply:
AFAICS it represents a page that has the Attachments tab (``AttachmentsPane``) and knows how to open / activate it, either by mouse click or by keyboard shortcut. You think those could be moved as static methods in ``AttachmentsPane``? I'm not sure it's better.

Reply:
I oscillated between having a new class and using static methods, I'm not sure which is the better option